documentation/release_notes.markdown

```diff
@@ -4,116 +4,104 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
-[static catalogs]: https://docs.puppet.com/puppet/4.4/reference/static_catalogs.html
-[file resources]: https://docs.puppet.com/puppet/4.4/reference/types/file.html
-[Puppet catalogs]: https://docs.puppet.com/puppet/4.4/reference/subsystem_catalog_compilation.html
-[environment classes API]: ./puppet-api/v3/environment_classes.markdown
-[classes]: https://docs.puppet.com/puppet/latest/reference/lang_classes.html
-[resource type API]: https://docs.puppet.com/puppet/latest/reference/http_api/http_resource_type.html
+[Trapperkeeper]: https://github.com/puppetlabs/trapperkeeper
+[service bootstrapping]: ./configuration.markdown#service-bootstrapping
+[auth.conf]: ./config_file_auth.markdown
 
-## Puppet Server 2.3.2
+## Puppet Server 2.4
 
-Released April 27, 2016.
+Released May 19, 2016.
 
-This is a bug-fix release that also resolves a security issue in endpoint URL decoding.
+This is a feature and bug-fix release of Puppet Server that also upgrades its included [Trapperkeeper][] framework from version 1.3.1 to 1.4.0.
 
-### **Security fix:** CVE-2016-2785 - Correctly decode endpoint URLs
+This release also adds packages for Ubuntu 16.04 LTS (Xenial Xerus) and no longer includes packages for Fedora 21, which reached its end of life in December.
```

Comment:
I think it also adds packages for Ubuntu 15.10 (Wily Werewolf)? I think we didn't have those before. But may want to confirm with @mckern or @underscorgan.